.gitignore

```
@@ -8,3 +8,5 @@
 .externalNativeBuild
 .cxx
 local.properties
+
+.kotlin/errors/
```

```suggestion
.kotlin
```
I'm tempted to suggest this, but I'm not sure if there are things inside .kotlin that we want to submit to the repository